helper/schema/schema_test.go

```diff
@@ -3947,6 +3947,33 @@ func TestSchemaMap_Validate(t *testing.T) {
 			Err: false,
 		},
 
+		"Good sub-resource, interpolated value": {
```

Comment:
Sorry, started typing before I saw your update... 
That looks right, and I swear I had it fail when I tested it. Let me see what's going on.